requirements.txt

```diff
@@ -0,0 +1,2 @@
+imapclient==2.2.0
```

Comment:
The only thing I worry about is the long-term maintenance. For a project like this that isn't too specific in bleeding-edge feature usage, I'm happy to just leave the version numbers off and let it install latest.